src/app/public/modules/inline-form/fixtures/inline-form.fixture.module.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  CommonModule
+} from '@angular/common';
+
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  SkyAppWindowRef
+} from '@skyux/core';
+
+import {
+  SkyInlineFormAdapterService
+} from '../inline-form-adapter.service';
+
+import {
+  SkyInlineFormFixtureComponent
+} from './inline-form.fixture';
+
+import {
+  SkyInlineFormModule
+} from '../inline-form.module';
+
+@NgModule({
+  declarations: [
+    SkyInlineFormFixtureComponent
+  ],
+  imports: [
+    CommonModule,
+    SkyInlineFormModule
+  ],
+  providers: [
+    SkyInlineFormAdapterService,
```

Comment:
Shouldn't need to provide the adapter service if it's provided by `SkyInlineFormModule`?

Reply:
Correct. This is removed.